templates/extension-template-js/index.js

```diff
@@ -26,12 +28,24 @@ var _globalName_ = (function (jspsych) {
 
     on_finish(params) {
       return {
-        data_property: "data_value",
+        data_name: 99, // Make sure this type and name matches data_name
+        data_name2: "hello world!", // Make this this type and name matches data_name2
       };
     }
   }
   ExtensionNameExtension.info = {
     name: "{name}",
+    version: version,
+    data: {
+      /** This comment will be scraped as metadata for data_name when running the metadata module.  */
```

Comment:
It will also be part of documentation eventually. Maybe provide this as instructions?

```
/** Provide a clear description of the paramater that could be used as documentation. We will eventually use these comments to automatically build documentation and produce metadata. */
```

Reply:
And this could be added to the trial parameters as well

Reply:
Fixed for parameters & data of plugin and data of extensions.